apps/web-giddh/src/app/shared/header/components/share-account-modal/share-account-modal.component.ts

```
@@ -32,14 +42,25 @@ export class ShareAccountModalComponent implements OnInit, OnDestroy {
-    constructor(private store: Store<AppState>, private accountActions: AccountsAction) {
+    constructor(private store: Store<AppState>, private accountActions: AccountsAction, private router: Router, private groupWithAccountsAction: GroupWithAccountsAction, private settingsProfileActions: SettingsProfileActions) {
         this.activeAccount$ = this.store.pipe(select(state => state.groupwithaccounts.activeAccount), takeUntil(this.destroyed$));
         this.activeAccountSharedWith$ = this.store.pipe(select(state => state.groupwithaccounts.activeAccountSharedWith), takeUntil(this.destroyed$));
         this.allPermissions$ = this.store.pipe(select(state => state.permission.permissions), takeUntil(this.destroyed$));
+        this.activeCompany$ = this.store.pipe(select(state => state.settings.profile), takeUntil(this.destroyed$));
     }
 
     public ngOnInit() {
+        this.activeCompany$.pipe(takeUntil(this.destroyed$)).subscribe(activeCompany => {
+            if (activeCompany) {
+                if (activeCompany?.subscription?.planDetails?.restrictedModules.hasOwnProperty(this.restrictedModules.Users) && activeCompany?.moduleRestrictionStatus) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)